.travis.yml

```diff
@@ -2,4 +2,4 @@ sudo: false
 language: ruby
 rvm:
   - 2.5.1
-before_install: gem install bundler -v 1.16.1
+before_install: gem install bundler -v 2.0.1
```

Comment:
Is this the same version of bundler that Netlify uses?

Reply:
all of our other projects seem to have upgraded to 2.0.1 so I would assume

https://github.com/crdschurch/crds-media/blob/576d70d1b8d6aefe62ebb177c436367ebac10899/Gemfile.lock#L257